netci.groovy

```diff
@@ -195,7 +195,7 @@ set TMP=%TEMP%
     }
   }
 
-  def triggerPhraseOnly = true
+  def triggerPhraseOnly = false
   def triggerPhraseExtra = "open-vsi"
```

Comment:
Since this is being made a bit more official, can we make the name more generally understandable? Per our other naming, windows_debug_integration? Already had once instance today of somebody asking "what does VSI mean?", and 'open' isn't terribly descriptive.